src/main/java/io/github/syst3ms/skriptparser/expressions/ExprDefaultValue.java

```diff
@@ -26,8 +25,7 @@
             Object.class,
             false,
             3,
-            "%object% \\|\\| %object%",
```

Comment:
Why remove this pattern?

Reply:
Because `||` should be logical OR. Even when you ignore the potential parsing problems, this is really counter-intuitive.